Lint-Whoppers/nutshell
# Jb task features1

# Description

I have merged everyones code with mine and now should be able to see my html element as well as everyone elses at once when you run npm start. Shouldn't be any merge conflicts should be smooth sailing.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ x] New feature (non-breaking change which adds functionality)

# Testing Instructions

Pull down the branch, run NPM start from the root and if you see all four boxes populated then we are good to go and merge!



# Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ x] I have added test instructions that prove my fix is effective or that my feature works
